platform/kube/inject/testdata/auth.cert-dir.yaml.injected

```diff
@@ -49,10 +49,16 @@ spec:
         securityContext:
           runAsUser: 1337
         volumeMounts:
+        - mountPath: /etc/istio
```

Comment:
Any chance we could combine the istio config and the certs in the same volume ? And under /home/istio ?